StandardLibrary/runtimes/go/ImplementationFromDafny-go/UUID/externs.go

```diff
@@ -0,0 +1,36 @@
+package UUID
+
+import (
+	"fmt"
+
+	"github.com/dafny-lang/DafnyRuntimeGo/v4/dafny"
+	"github.com/dafny-lang/DafnyStandardLibGo/Wrappers"
+	"github.com/google/uuid"
+)
+
+func ToByteArray(seq dafny.Sequence) Wrappers.Result {
+	var s string
+	for i := dafny.Iterate(seq); ; {
+		val, ok := i()
+		if ok {
+			s = s + string(val.(dafny.Char))
+		} else {
+			break
```

Comment:
question: is it OK to ignore not-ok return here?

Reply:
The way `dafny.Iterate()` works is that it returns not okay when the end of sequence has been reached - and that's when we break.

Reply:
Given this would it be better to change the variable name to say, `more`?

Reply:
Named is as `notEndOfSequence`.